commons-search/src/main/java/org/exoplatform/commons/search/es/ElasticSearchServiceConnector.java

```diff
@@ -193,7 +193,7 @@ protected String buildFilteredQuery(String query, Collection<String> sites, List
     if (StringUtils.isNotBlank(query)) {
       List<String> queryParts = Arrays.asList(query.split(" "));
```

Comment:
By applying this change :
https://github.com/Meeds-io/commons/blob/e98dc794a0e89bb688ec0c00c9171169307dd3ca/commons-search/src/main/java/org/exoplatform/commons/search/es/ElasticSearchServiceConnector.java#L196

The fuzzy search will not be applied anymore.

I suggest to change this line:
https://github.com/Meeds-io/commons/blob/e98dc794a0e89bb688ec0c00c9171169307dd3ca/commons-search/src/main/java/org/exoplatform/commons/search/es/ElasticSearchServiceConnector.java#L194
into : 
`
      List<String> queryParts = Arrays.asList(query.split("\\+\\-=\\&\\|><\\!\\(\\)\\{\\}\\[\\]\\^\"\\*\\?:\\/ @"));
`
This way you will delete special characters from searched words.
PS: we can't apply unfortunately use `whitespace` `analyzer` on query search (See https://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-dsl-query-string-query.html)

Reply:
Done ! Thanks